File: 0229-majority-element-ii/0229-majority-element-ii.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        my_map = {}
        li = []
        for num in nums :
            if num in my_map :
                my_map[num] += 1
            else :
                my_map[num] = 1
        
        for key,val in my_map.items() :
            if val > len(nums) / 3:
                li.append(key)
                
        return li

File: 0229-majority-element-ii/0229-majority-element-ii.java
import java.util.Map.Entry;
class Solution {
    public List<Integer> majorityElement(int[] nums) {
        HashMap<Integer,Integer> myMap = new HashMap<>();
        
        for(Integer num : nums){
            if(myMap.containsKey(num)){
                myMap.put(num,myMap.get(num) + 1);
            }
            else{
                myMap.put(num,1);
            }
        }
        
        List<Integer> ans = new ArrayList<>();
        
        for(Entry<Integer,Integer> entry : myMap.entrySet()){
            if(entry.getValue() > (nums.length / 3)){
                ans.add(entry.getKey());
            }
        }
        
        return ans;
    }
}